src/addrman.h

```diff
@@ -136,6 +159,15 @@ class AddrMan
     void SetServices(const CService& addr, ServiceFlags nServices);
 
     const std::vector<bool>& GetAsmap() const;
+
+    /*
+     * Test-only function
+     *
+     * Finds the address record in AddrMan and returns information about its
```

Comment:
Use the indicative mood:

Find the address record in AddrMan and return information about its